assets/js/hooks/useViewOnly.js

```diff
@@ -37,6 +39,8 @@ export default function useViewOnly() {
 
 	return (
 		viewContext === VIEW_CONTEXT_MAIN_DASHBOARD_VIEW_ONLY ||
-		viewContext === VIEW_CONTEXT_ENTITY_DASHBOARD_VIEW_ONLY
+		viewContext === VIEW_CONTEXT_ENTITY_DASHBOARD_VIEW_ONLY ||
+		viewContext === VIEW_CONTEXT_WP_DASHBOARD_VIEW_ONLY ||
+		viewContext === VIEW_CONTEXT_ADMIN_BAR_VIEW_ONLY
 	);
```

Comment:
Since this check has grown, with potential to expand in the future, maybe it can be converted to array check. Like putting all `VIEW_CONTEXT_*` constants into an array, like `contexts = [ VIEW_CONTEXT_MAIN_DASHBOARD_VIEW_ONLY, ... ]` and then in `return` checking only for `contexts.includes( viewContext )`. What do you think?

Reply:
Currently, it looks good now with only four view contexts. It will not grow in the near future. We can refactor it if it's more than five. WDYT?

Reply:
Sure, it is fine either way, if it grows more at some point we can consider refactoring it then. Thanks